dev/conductor/lib/globals.dart

```diff
@@ -7,6 +7,8 @@ import 'package:file/file.dart';
 import 'package:file/local.dart';
 import 'package:platform/platform.dart';
 
+import './proto/conductor_state.pb.dart' as pb;
```

Comment:
```suggestion
import 'proto/conductor_state.pb.dart' as pb;
```
Or is there some reason to `./`?

Reply:
hmm, no reason, done